gacl/docs/translations/russian/manual_rus.html

```diff
@@ -108,7 +108,7 @@ <h2>Понимание контроля действия</h2>
     </tr>
 </table>
 <p>Колонки показывают список комнат, к которым Хан может ограничить доступ, а ряды показывают персон, которые могли бы запросить доступ к тем или иным комнатам. Более широкое объяснение:
-<p>“комнаты” - это вещи (объекты) для управления доступа к ним. Мы называем эти объекты <strong>ACO</strong> (<em>Access Control Objects</em> - Объекты контроля доступа);
+<p>“комнаы” - это вещи (объекты) для управления доступа к ним. Мы называем эти объекты <strong>ACO</strong> (<em>Access Control Objects</em> - Объекты контроля доступа);
```

Comment:
This looks like a translation encoding issue

Reply:
wouldn't worry about this